3d_segmentation/swin_unetr_btcv_segmentation_3d.ipynb

```diff
@@ -461,15 +331,13 @@
     "    cache_rate=1.0,\n",
     "    num_workers=8,\n",
     ")\n",
-    "train_loader = DataLoader(\n",
-    "    train_ds, batch_size=1, shuffle=True, num_workers=8, pin_memory=True\n",
-    ")\n",
+    "train_loader = ThreadDataLoader(train_ds, num_workers=0, batch_size=1, shuffle=True)\n",
     "val_ds = CacheDataset(\n",
     "    data=val_files, transform=val_transforms, cache_num=6, cache_rate=1.0, num_workers=4\n",
     ")\n",
-    "val_loader = DataLoader(\n",
-    "    val_ds, batch_size=1, shuffle=False, num_workers=4, pin_memory=True\n",
-    ")"
+    "val_loader = ThreadDataLoader(val_ds, num_workers=0, batch_size=1)\n",
+    "\n",
+    "set_track_meta(False)"
```

Comment:
Please 1 line comment here to help users understand the logic why we set it after caching.

Thanks.